test/module_test.ts

```diff
@@ -62,23 +62,33 @@ describe('library name', () => {
   var transpiler: main.Transpiler;
   var modTranspiler: ModuleTranspiler;
   beforeEach(() => {
-    cwd = process.cwd();
-    transpiler =
-        new main.Transpiler({failFast: true, generateLibraryName: true, basePath: cwd + '/a'});
-    modTranspiler = new ModuleTranspiler(transpiler, new FacadeConverter(transpiler), true);
+    transpiler = new main.Transpiler({failFast: true, generateLibraryName: true, basePath: 'a'});
+    modTranspiler = new ModuleTranspiler(transpiler, new FacadeConverter(transpiler), true, null);
   });
   it('adds a library name', () => {
     var results = translateSources(
-        {'a/b/c.ts': 'var x;'}, {failFast: true, generateLibraryName: true, basePath: cwd + '/a'});
+        {'a/b/c.ts': 'var x;'}, {failFast: true, generateLibraryName: true, basePath: 'a'});
```

Comment:
I had missed cleaning it up in the last change. Cleaned it up while I add some tests for this change.